chocolatey/choco
# (#3364) Skip unusable tab completion tests

## Description Of Changes

Skip tests that are not applicable to the running PowerShell version.

## Motivation and Context
These tests are verifying the old method and are targeting an implementation detail, rather than actually checking the tab completion results, which the following tests do anyway.

As such, we're best off skipping them as there is no direct way to check the same things for the Register-ArgumentCompleter method. The tests for the tab completion results themselves covers this adequately already.

## Testing

1. Run `.\Invoke-Tests.ps1 -Tag Profile`
2. Ensure tests flagged to be skipped for tab completion are skipped in PowerShell 5 and above.

### Operating Systems Testing

Win10

## Change Types Made



## Change Checklist


## Related Issue

#3364
